docs/examples/imagestreams/README.md

```diff
@@ -0,0 +1,20 @@
+# Image Streams
+
+An image stream can be used to automatically perform an action, such as updating a deployment, when a new image, such as a new version of the base image that is used in that deployment, is created.
+
+Like the way most of the Kedge constructs are, for `imageStreams`, the ImageStreamSpec and ObjectMeta have been merged at the same YAML level.
+
+A valid OpenShift `ImageStream` resource can be specified at the root level of the Kedge spec in a field called `imageStreams` like we see in [is.yml](is.yml):
+
+```yaml
+name: webapp
+imageStreams:
+- tags:
+  - from:
+      kind: DockerImage
+      name: centos/httpd-24-centos7:latest
```

Comment:
The example is technically right, but pulling "latest" tag as "2.4" is not something that users should do. Can we change it to "2.4"? (the same thing in other examples)